packages/storybook/.storybook/main.js

```diff
@@ -15,8 +15,7 @@ module.exports = ({ args }) => {
       '../../../plugins/search/src/**/*.stories.tsx',
     ];
   } else {
-    const rootDir = path.resolve(__dirname, '../../..');
-    stories = args.map(arg => path.join(rootDir, arg, 'src/**/*.stories.tsx'));
+    stories = args.map(arg => `../../../${arg}/src/**/*.stories.tsx`);
```

Comment:
Does ``path.posix.join(`../../..`, arg, 'src/**/*.stories.tsx')`` work fine too? Feeling that clearly communicates that we always want posix-style paths

Reply:
Updated to explicitly use `posix` paths.